content/docs/comparison/infrastructure-as-code.md

```diff
@@ -5,13 +5,15 @@ type: docs
+These tools often manage resources such as virtual machines and software deployments. A common pattern used in describing these resources is "cattle vs. pets". Cattle refers to components that are easy to replace and where state is not important. This includes web services, APIs, and jobs. Pets refer to components that are difficult to replace or update, including databases. Users must be careful not to lose data when updating or replacing these types of software.
 
-Operators reflect this reality by taking responsibility for the software that they run. They do this in two ways - providing a declarative shell called a Custom Resource Definition (CRD) that creates an API for operator users to define the desired state of their software, as well as a controller that continuously asserts that these resources are always advancing to the desired state.
+IaC tools often consider the resources running these two types of software to be similar. Stateful applications change over time and drift from how IaC deployed them. Terraform and Ansible only make changes when running them, adding even more risk of drift. Kubernetes resources drift as well. For example, the scheduler replaces pods when they crash or their host dies. Admission webhooks or other controllers change these resources based on cluster requirements.
 
-IaC tools work well in this paradigm. Instead of using Terraform to work with the underlying components that make up a complex software deployment, use Terraform or Ansible to deploy the KUDO controller, and the custom resources that define the operator for the Kubernetes application you want to deploy. This works for any operator, combining the single-view infrastructure convergence power of IaC tooling with the long-term maintenance capabilities of Operators.
+Operators reflect this reality by taking responsibility for the software that they run. Operators are two combined parts: a custom resource and a controller. The custom resource contains the declarative information for the managed application. Controllers ensure that the resources in the cluster and custom resource match.
+
+IaC tools work well in this paradigm. Do not use basic Kubernetes like pods and deployments for pets. Terraform and Ansible can create the controller and the custom resource. This enables IaC tools to manage the controller as cattle, and the controller to manage the pet.
```

Comment:
This paragraph feels a bit disconnected to me. The second sentence is a bit bold for me, i feel the previous version was nicer to read.

Reply:
reverted that part